packages/gcf-utils/README.md

```diff
@@ -4,6 +4,22 @@
 
 ## Usage
 
+### GCFLogger
+
+`GCFLogger` is a [Pino](https://github.com/pinojs/pino)-based logger with added configurations for Google Cloud Functions. Note that this logger is a singleton instance shared across all modules.
+
+```
+import {GCFLogger} from 'gcf-utils';
+
+let logger = GCFLogger.get();
```

Comment:
I'm surprised you're allowed to even have a method named `get` in JavaScript!  From a usability perspective, I'd rather have:

```ts
import {getLogger} from 'gcf-utils';
const logger = getLogger();
```

Reply:
Oops! Didn't think of that. I'm not super familiar with JavaScript conventions. In Java world this would be `getInstance()` so I think `getLogger()` makes sense. I'll fix it - thanks for the catch!